.github/workflows/tool_cleanup.yml

```diff
@@ -0,0 +1,161 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Reusable workflow that handles tool request cleanup.
+name: 'aod-tool-cleanup'
+
+# Support below trigger:
+# pull_request:
+#   types: 'closed'
+#   paths: 'tool.yaml'
```

Comment:
Is this intentional? Or should live somewhere else?

Reply:
Removing.